client/src/SubscriptionSelection.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: "Inter-Thin";
+  src: url("Inter-Thin.ttf");
+}
```

Comment:
Ah, cool! That's good to know you can do this :) Creating your own font-face by giving it a name and then a url for where the font can be found.

Reply:
Yes! I learned something new here too :)